cool-sbom-template.spdx.json

```diff
@@ -0,0 +1,369 @@
+{
+  "SPDXID": "SPDXRef-DOCUMENT",
+  "spdxVersion": "SPDX-2.3",
+  "dataLicense": "CC0-1.0",
+  "name": "Collabora Online",
+  "documentNamespace": "http://spdx.org/spdxdocs/coolwsd-24.04.10.3",
+  "creationInfo": {
+    "creators": [
+      "Tool: Custom Script"
+    ],
+    "created": "2024-12-15T12:00:00Z"
+  },
+  "packages": [
+    {
+      "SPDXID": "SPDXRef-coolwsd",
+      "name": "coolwsd",
+      "versionInfo": "24.04.10.3",

```

Comment:
Nit: you process this template with a script, so I assume this value is a placeholder, but then it would be useful to set it to something like @VERSION@ to make it clear it will be replaced.